public/app/plugins/panel/table/transformers.ts

```diff
@@ -6,6 +6,33 @@ import { TableTransform } from './types';
 import { Column, TableData } from '@grafana/data';
 
 const transformers: { [key: string]: TableTransform } = {};
+const timeSeriesReducer = (data: any): any[] => {
```

Comment:
is it really a reducer? Feels more like a filter of sorts? a reducer I would expect to take in an array of stuff and return a single value (I guess redux reducers to something else).

Reply:
Yeah, `dataFormatFilterer`, `timeSeriesFilterer` and `tableFilterer` might be better naming.